### ISSUE TITLE
CHelper in Intellij doesn't pick up the data

### ISSUE THREAD

**Issue by mafulafunk** [jmerle/competitive-companion#82]
The companion dumps this in debug mode. But the chelper plugin in intellij doesn't pick it up.
What is the next place to look for further analysis?
<img width="1273" alt="Screenshot 2020-04-16 at 23 04 00" src="https://user-images.githubusercontent.com/99756/79506860-fbd74580-8036-11ea-8d0e-e5886dd1c89d.png">

**Comment by jmerle** [comment]
That's odd. Does the Event Log in IntelliJ IDEA show anything when attempting to parse a problem?

**Comment by mafulafunk** [comment]
It does: 🙃
<img width="470" alt="Screenshot 2020-04-16 at 23 42 13" src="https://user-images.githubusercontent.com/99756/79509546-021bf080-803c-11ea-9f89-7ab7e07904fb.png">

**Comment by mafulafunk** [comment]
But as far as I can tell, neither the Chelper nor the Chrome Plugin was changed.

**Comment by mafulafunk** [comment]
Maybe this helps: A copy and Paste of the Json dump in the console:
{
    "name": "B. Badge",
    "group": "Codeforces - Codeforces Round #503 (by SIS, Div. 2)",
    "url": "https://codeforces.com/problemset/problem/1020/B",
    "interactive": false,
    "memoryLimit": null,
    "timeLimit": null,
    "tests": [
        {
            "input": "3\n2 3 2\n",
            "output": "2 2 3 \n"
        },
        {
            "input": "3\n1 2 3\n",
            "output": "1 2 3 \n"
        }
    ],
    "testType": "single",
    "input": {
        "fileName": "input",
        "type": "file"
    },
    "output": {
        "fileName": "output",
        "type": "file"
    },
    "languages": {
        "java": {
            "mainClass": "Main",
            "taskClass": "BBadge"
        }
    }
}

**Comment by mafulafunk** [comment]
https://github.com/EgorKulikov/idea-chelper/issues/81

**Comment by jmerle** [comment]
Aha, found the issue. Apparently Codeforces' problem pages were updated slightly, due to which the time limit and memory limit get set to `null` when parsing some problems. This is a Competitive Companion bug which probably causes a `NullPointerException` somewhere in CHelper. Will fix it in a moment, although I can't release it immediately because the extension is currently Pending Review on the Chrome Web Store due to a change in the description that I made, and I can't upload a new version until that is done.
